src/Sarif/Schemata/sarif-schema.json

```diff
@@ -1705,7 +1707,7 @@
         },
 
         "frames": {
-          "description": "An array of stack frames that represent a sequence of calls, rendered in reverse chronological order, that comprise the call stack.",
+          "description": "An array of stack frames that represents a sequence of calls, rendered in reverse chronological order, that comprise the call stack.",
```

Comment:
>represents [](start = 56, length = 10)

Thanks.

Reply:
Although I suppose you could argue whether the word modifies "array" or "stack frames".

---
In reply to: [226731080](https://github.com/Microsoft/sarif-sdk/pull/1114#discussion_r226731080) [](ancestors = 226731080)

Reply:
By making it singular, there is no argument. :) There was no argument before, it is just my sense that this form is less likely to be construed as an error. :)

---
In reply to: [226731287](https://github.com/Microsoft/sarif-sdk/pull/1114#discussion_r226731287) [](ancestors = 226731287,226731080)